### ISSUE TITLE
xrefitems not listed if the page is referenced multiple times

### ISSUE THREAD

**Issue by enen92** [doxygen/doxygen#6580]
In our documentation, we document some values (arrays) in the middle of cpp code since they are more easy to track. We group them under sections of the same page, as illustrated here:
https://codedocs.xyz/xbmc/xbmc/modules___general___list_of_gui_access.html

We've now started to track changes to those values using `\xrefitems` (one per release version) in order to provide a changelog. The problem is, they are not being listed.

A simple example:

**alias**
`ALIASES="custom_change=\xrefitem custom_change \"This is a custom change\" \"\""`

**test.cpp** file:

```
/// @page test TEST Page
/// \tableofcontents
///
/// \section test_description Description
/// this is some description
/// @custom_change a mark in description (this is only for testing purposes)
///

this is some cpp code

/// @page test
/// @section section1 Section 1
/// some text
///
/// @custom_change a mark in section1 (we want this one)
///

this is some more cpp code (values for section 1)

/// @page test
/// @section section2 Section 2
/// some text
///
/// @custom_change a mark in section 2 (we want this one)

yet more cpp code (values for section 2)
```

After running doxygen, the only change that appears in the `custom_change` page is the one mentioned in the description section. Probably because it is the first time `@page` is referred. All the others are just ignored.

Thanks a lot in advance

**Comment by enen92** [comment]
Thank you very much @albert-github. I'll compile and test now. I can't believe it was such a simple fix (I spent some time trying to fix it myself before opening the issue). By the way, does doxygen have any release cycle? Looks like the last release was almost 1 year ago. No rush, just curious.
Regards

**Comment by enen92** [comment]
Compiled and run tested. Confirmed #14549 fixes the issue! I'll close the issue.
Thanks very much for dedicating your time to fix this one.
Best regards

**Comment by albert-github** [comment]
Please do not close the issue (will be done when a release is done), please reopen.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.15.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
